woocommerce/wc-admin
# Use snackbar before CES modal

Fixes #5390

The CES modal/prompt needs to be triggered by a snackbar. This adds the snackbar to the `CustomerEffortScoreTracks` component.

![image](https://user-images.githubusercontent.com/224531/96204307-53834900-0fa7-11eb-816f-60eb25aa04df.png)

### Detailed test instructions:

- Add a new product
- The snackbar should appear
- Click "Give feedback"
- The CES modal should appear

### Changelog Note:

Dev: Use snackbar before CES modal
